src/Agent/NewRelic/Agent/Core/AgentHealth/HealthCheck.cs

```diff
@@ -49,12 +49,12 @@ public void TrySetHealth((bool IsHealthy, string Code, string Status) healthStat
             }
         }
 
-        public string ToYaml()
+        public string ToYaml(string entityGuid)
```

Comment:
All the other health check information is part of the health check object. It might be better to make the entity guid part of the health check object itself.